Adding a new updated bot for Voiceless Voice

owenshi2 opened the pull request:
My attempt at programming a modern deck. Checked the documentation but had some issues with running windbot after adding the executors. Please let me know if there's any errors with the formatting or bot itself encountered as I haven't been able to run or test the bot locally.

owenshi2 committed:
Added Voiceless Voice Bot AI

owenshi2 committed:
Added Voiceless Voice AI Bot

owenshi2 committed:
Added Voiceless Voice AI Bot

owenshi2 committed:
Update bot.conf for Voiceless

mercury233 committed:
fix proj

mercury233 committed:
Merge branch 'master' into patch-voiceless

mercury233 commented:
As you can see, the code you posted are incomplete.

owenshi2 commented:
Thank you for the fixes, mercury! Is this version able to be run in the bot? If so, do I copy the files from the repo into the EDOPro Windbot folder to test it?

Wind2009-Louse commented:
There's still many errors in scripts. Even if grammar errors are fixed, there are still many cards that cannot be activated. Like:
Prayers of the Voiceless Voice
Trias Hierarchia
and so on

owenshi2 commented:
Hi Louse,

Thank you for taking a look at my code! How do you suggest I correct the issues brought up?
If I remove the cards mentioned from being able to be activated altogether, would the bot figure it out when given the prompt?

Wind2009-Louse commented:
> Hi Louse,
> 
> Thank you for taking a look at my code! How do you suggest I correct the issues brought up? If I remove the cards mentioned from being able to be activated altogether, would the bot figure it out when given the prompt?

You can check https://github.com/IceYGO/windbot/wiki/How-to-create-your-WindBot-AI.
Bot will activate a card only when you defind it with ``AddExecutor(ExecutorType.Activate, _id)``, or with ``AddExecutor(ExecutorType.Activate)``(if you use the latter one, bot will activate any card it can do, so it's not recommended).

owenshi2 committed:
Updated Voiceless Executor

Fixed some cards not activatable.

owenshi2 committed:
Removed woman

Got rid of IP Masquerena since the script would be too complicated to implement for me. Perhaps in the future?

owenshi2 commented:
Ok; I checked which cards don't have an activated and added some functions for the unique ones. Do let me know if there's other things to be fixed!


(I also really liked how in-depth the activation functions for SP and Chaos Angel from the Lab Executor you made, Louse, so I added some of the functions in as well; hope it's alright)
